server/src/test/java/org/eclipse/lsp/cobol/usecases/TestReplaceCompilerDirective.java

```diff
@@ -89,6 +92,29 @@
           + "           MOVE 0 to {$PAYROLL-WEEK}.\n"
           + "           MOVE 0 to {$PAYROLL-GROSS-ID}.";
 
+  private static final String TEXT5 =
```

Comment:
Have added the same text which caused an error in the demo. I am not sure how to cover this bug in the use-case test.
Not added any assert on the returned analysis (line no -167 below) as the use-case asserts those.

Reply:
If I understood correctly, the problem was with a missing location for some variable. All the locations are asserted by the use case engine, so this test is enough.